RadixWallet/Features/TransactionReviewFeature/TransactionReviewAccount/TransactionReviewAccount+View.swift

```diff
@@ -96,134 +93,64 @@ extension TransactionReviewAccount {
 
 // MARK: - TransactionReviewResourceView
 struct TransactionReviewResourceView: View {
-	let transfer: TransactionReview.Transfer
+	let transfer: ResourceBalance // FIXME: GK use viewstate
```

Comment:
FIXME comment still relevant?

Reply:
Yes, this should ideally be further refactored, just a little bit. I really want to use the ViewState version here, but it's not urgent.